packages/core/scripts/local/getMedianHistoricalPrice.js

```diff
@@ -15,7 +15,9 @@ require("dotenv").config();
 
 const UMIP_PRECISION = {
```

Comment:
Its better actually to define these in [this file](https://github.com/UMAprotocol/protocol/blob/master/packages/common/src/PriceIdentifierUtils.js) because this is imported into the Liquidator/Disputer bots. 

This file `getMedianHistoricalPrice` should actually be updated to import from that file as well.

Reply:
Thanks, @nicholaspai, I'll go on and take care of updating both of those 👍🏾

Reply:
Hey @jaxrtech actually I'm mistaken. There is a difference between the precision used for UMIP's and for the identifiers. Will the BTCDOM ETHDOM be OK to resolve to 18 decimal precision?

Reply:
@nicholaspai Should commit 8c940bceb5e4b1e41003d96d6b0f3c381eb06466 be reverted then?

Reply:
yes. that change needs to be rolled back. In particular, you should not set `IDENTIFIER_NON_18_PRECISION` within `packages/common/src/PriceIdentifierUtils.js` to 2. This will scale your feeds incorrectly. This number `IDENTIFIER_NON_18_PRECISION` is used to scale values returned from the DVM to accommodate non-18 decimal delimitation. For example, an identifier denominated in BTC will be scaled by 1e8. As your collateral type is Dai, you don't want to make this change & it should default to 18.

Reply:
@chrismaree @nicholaspai  Makes sense, I've reverted the commit.